src/pages/install/macos.md

```diff
@@ -0,0 +1,52 @@
+
+<hgroup>
+
+← [Back to install instructions](/install)
+
+# Installing RGBDS on macOS
+
+</hgroup>
+
+RGBDS is available on [Homebrew](https://brew.sh) as the [`rgbds` package](https://formulae.brew.sh/formula/rgbds).
+
+```bash
+brew install rgbds
+```
+
+:::caution
+
+This may not be suitable if your version of macOS is too old for Brew to support it.
+(In particular, this leads to very long installation times.)
```

Comment:
Brew always distributes the "recipes", but only distributes binary packages ("bottles") to supported OSes.

Reply:
So the installation method is definitely "supported" with brew, it's just that you don't get a binary because the maocs version is not "supported". I would be careful and avoid implying that not supported means impossible or not feasible

Reply:
It is supported, but not suitable. The Brew package, without binaries, ends up compiling **rustc** from source, which tends to take days. This has already tripped several people (last of which being "Rainbow Metal Pigeon" on Discord, if you want to look that up), who were thus going to stick to older versions; so the warning box is absolutely warranted.

Reply:
Might be worth explaining that?

Reply:
Then something like:

```suggestion
Installation via Brew may not be advisable for users on older macOS versions, since binaries are only available for [recent macOS major releases](https://formulae.brew.sh/formula/rgbds). Significantly prolonged compilation times are expected on older macOS systems.
```

Reply:
The link immediatly shows which macos releases have bottles (binaries) available

Reply:
I've incorporated a slightly amended version of Antonio's above suggestion in https://github.com/gbdev/rgbds-www/pull/60/commits/abd5c8ff5a4a09763ca3058fba91011f48bcf6c6. Is it satisfactory?